go-controller/pkg/clustermanager/clustermanager_test.go

```diff
@@ -1236,6 +1237,16 @@ var _ = ginkgo.Describe("Cluster Manager", func() {
 							return fmt.Errorf("transit switch ips for node %s not allocated", n.Name)
 						}
 
+						_, transitSwitchSubnet, err := net.ParseCIDR(config.ClusterManager.V4TransitSwitchSubnet)
+						if err != nil {
+							return err
```

Comment:
nit: make the error wrapped meaningfully "fmt.Errorf("could not parse... blah %v", err)"

Reply:
Ack